src/index.ts

```diff
@@ -27,20 +27,21 @@ loadCommands(foldersPath, client);
 // Log in to Discord with your client's token
 client.login(token);
 
-function loadCommands(foldersPath: string, client: BotClient) {
-  const commandFolders = fs.readdirSync(foldersPath);
+function loadCommands(folderPath: string, botClient: BotClient) {
+  const commandFolders = fs.readdirSync(folderPath);
 
   for (const folder of commandFolders) {
-    const commandsPath = path.join(foldersPath, folder);
+    const commandsPath = path.join(folderPath, folder);
     const commandFiles = fs.readdirSync(commandsPath).filter(file => file.endsWith('.js'));

```

Comment:
It's a minor thing but I think that it adds to readability if you add a line break after each method call in the chain, especially if the chain is longer than in this example. 
e.g. --> 
```javascript
fs.readdirSync(commandsPath).
  filter(file => file.endsWith('.js'));
```